griptape/tools/google_docs/tool.py

```diff
@@ -0,0 +1,270 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING
+import logging
+from attr import field, define
+from schema import Schema, Optional, Literal
+from griptape.artifacts import ErrorArtifact, InfoArtifact, BlobArtifact, ListArtifact
+from griptape.utils.decorators import activity
+from griptape.tools import BaseGoogleClient
+
+if TYPE_CHECKING:
+    from googleapiclient.discovery import Resource
+
+@define
+class GoogleDocsClient(BaseGoogleClient):
+    DOCS_SCOPES = ["https://www.googleapis.com/auth/documents"]
+
+    DRIVE_SCOPES = ["https://www.googleapis.com/auth/drive.file"]
+
+    owner_email: str = field(default=None)
```

Comment:
1. Why does it have a default?
2. Why does it not have `kw_only`?

Reply:
Oops, I forgot to change this!